src/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead>
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Title</th>
+          <th scope="col">First name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">E-mail</th>
+          <th scope="col">Room No</th>
+          <th scope="col">Check-in Date</th>
+          <th scope="col">Check-out Date</th>
+          <th scope="col">Numer of Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((p, index) => {
+          return (
+            <tr key={index}>
```

Comment:
putting key here may not be the best way 
read more 
https://reactjs.org/docs/lists-and-keys.html

Reply:
I believe the problem is using index here as a key. So I assigned id numbers as keys.